index.js

```diff
@@ -1,11 +1,11 @@
-var fs = require('fs'),
-    path = require('path'),
-    through = require('through2'),
-    gutil = require('gulp-util'),
-    PluginError = gutil.PluginError,
-    exec = require('child_process').exec,
-    os = require('os'),
-    format = require('util').format;
+var fs = require("fs"),
+    path = require("path"),
+    through = require("through2"),
+    gutil = require("fancy-log"),
```

Comment:
I don't think this is correct. From the docs on `fancy-log`, there is no `log` method attached; rather, the main export _is_ the log function:

https://github.com/gulpjs/fancy-log/blob/master/index.js#L88-L92

Reply:
Fixed this with bf34a1d. Thank you for the suggestion